jsapp/js/projects/projectsTable/projectActions.module.scss

```diff
@@ -5,16 +5,19 @@
 .root {
   @include mixins.centerRowFlex;
   gap: sizes.$x10;
-}
 
-.menu {
-  @include mixins.floatingRoundedBox;
-  padding: sizes.$x6;
-  min-width: sizes.$x180;
+  // Override .k-button[disabled] styles to allow tooltips on disabled buttons
+  :global .k-button[disabled] {
+    pointer-events: auto; // Allow tooltips on disabled buttons
+    opacity: 1; // Make the tooltips fully opaque
 
-  // There is a `isFullWidth` property on Button component, but it also has text
-  // centering styles on it, so we can't use it.
-  :global .k-button {
-    width: 100%;
+    // Make the button appear non-pressable
+    .k-icon {
+      opacity: 0.5;
+    }
+    cursor: default;
+    &:active {
+      transform: translateY(0); // Don't depress button when clicked
+    }
```

Comment:
I was trying to test this on my mobile device and ran into some issues with my network security so I'm not 100% certain this shows on a physical device but using the emulator you can press the button and the box exists around it even though you cannot select. 
![Uploading Screen Shot 2023-08-25 at 2.47.26 PM.png…]()